k8s.io/configmap-nginx.yaml

```diff
@@ -252,6 +252,7 @@ data:
           rewrite ^/bounty$          https://github.com/kubernetes/kubernetes.github.io/issues?q=is%3Aopen+is%3Aissue+label%3ABounty redirect;
           rewrite ^/bot-commands$    https://prow.k8s.io/command-help.html redirect;
           rewrite ^/github-labels$   https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.md redirect;
+          rewrite ^/good-first-issue$ https://github.com/kubernetes/kubernetes/labels/good%20first%20issue redirect;
```

Comment:
Can we also include issues from projects in [kubernetes-sigs](https://github.com/kubernetes-sigs) ?